src/Avalonia.Controls/Platform/DragSource.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Reactive.Subjects;
+using System.Threading.Tasks;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.Input.DragDrop;
+using Avalonia.Input.DragDrop.Raw;
+using Avalonia.Input.Platform;
+using Avalonia.Input.Raw;
+using Avalonia.Threading;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Platform
+{
+    class DragSource : IPlatformDragSource
```

Comment:
So this is the "in-process" drag source that is used when there is no OS drag source available. It should probably be called `DefaultDragSource` or `InProcessDragSource` or something?